### ISSUE TITLE
Issues upgrading to A-Frame 1.4.1

### ISSUE THREAD

**Issue by diarmidmackenzie** [diarmidmackenzie/instanced-mesh#28]
Have been testing compatibility with A-Frame 1.4.1.  A few issues have come up...

Here's a list of the examples & tests that seem broken...

## Tests

**3d-models-multi-mesh**

Colors are wrong:
![image](https://user-images.githubusercontent.com/16045703/211152698-f0b059a7-26dc-4b83-a0c3-7918a5f67507.png)
(also other tests that use the same model have the same problem)

**env-map-example**

![image](https://user-images.githubusercontent.com/16045703/211152659-fe9ddd76-67e0-4284-9b8b-70cfe1f3b274.png)
Colors are wrong + instancing not happening.

Errors in console:
```
core:cubemap:warn <a-cubemap> did not contain exactly six elements each with a `src` attribute. 
13:30:33.677 browser.js:111 core:a-assets:warn Asset loading timed out in   3000 ms
13:30:33.690 browser.js:111 utils:src-loader:warn Selector "#env" does not point to <a-cubemap> 
```

**movement-multi-mesh**

hits error:
```
[GET http://127.0.0.1:8080/tests/framed-cell.js ](movement-multi-mesh.html:6          GET http://127.0.0.1:8080/tests/framed-cell.js net::ERR_ABORTED 404 (Not Found))
```
(maybe not a new issue?)
 
**pbr-example**

hits error:
```
http://127.0.0.1:8080/tests/assets/snowball/Snow_002_DISP.jpg 404 (Not Found)
```
(maybe not a new issue?)

## Examples

**animated-spheres**

![image](https://user-images.githubusercontent.com/16045703/211153091-b4bc86d1-7deb-48a4-a318-de406badc305.png)
Not working at all - likely the same issue as with `env-map-example`

**coloring-blocks**

Seems basically OK, but on deleting blocks, hit a few errors:
```
instanced-mesh.js:793 Member block-5-7-7 not found for modification
```
(maybe not a new issue, though I don't recall seeing it before)

**Comment by vincentfretin** [comment]
For 3d-models-multi-mesh, the wrong colors are because some textures are not created, the number of textures decreased from 29 to 10 because of this warning:
GLTFLoader.js:387 THREE.GLTFLoader: Unknown extension "KHR_materials_pbrSpecularGlossiness".
see https://github.com/mrdoob/three.js/wiki/Migration-Guide#146--147
Support for the KHR_materials_pbrSpecularGlossiness extensions has been removed from GLTFLoader and GLTFExporter. Older glTF assets should be losslessly converted from spec/gloss to metal/rough.
https://www.donmccurdy.com/2022/11/28/converting-gltf-pbr-materials-from-specgloss-to-metalrough/

**Comment by diarmidmackenzie** [comment]
Thanks, Vincent.

env-map issues are due to an A-Frame 1.4.1 problem: https://github.com/aframevr/aframe/issues/5230

I didn't look into the other problems yet.

**Comment by diarmidmackenzie** [comment]
I have now updated all examples to 1.4.2 and resolved most of the issues above - see #31

Only outstanding issue is the errors hit when deleting blocks in the coloring-blocks example, but that turns out to be a longstanding issue.  I'll raise a separate issue to track that.

**Comment by diarmidmackenzie** [comment]
See #32 for the unresolved issue.
